orders/views.py

```diff
@@ -42,20 +40,31 @@ def orders_history(request):
     to_return = []
 
     for order in last_orders:
-        pizzaOrders = order.pizzaorder_set.all()
-
-
+        pizza_orders = order.pizzaorder_set.all()
 
+        pizza_orders_treating = []
+        for pizza_order in pizza_orders:
+            pizza_orders_treating.append({"treated": False,
+                                          "content": pizza_order})
 
+        formatted_order = []
+        for pizza_order in pizza_orders_treating:
+            if not pizza_order['treated']:
+                pizza_order['treated'] = True
+                same_pizzas = []
+                for other_pizza_order in pizza_orders_treating:
+                    if not other_pizza_order['treated'] and other_pizza_order['content'].get_verbose_description() == pizza_order['content'].get_verbose_description():
+                        other_pizza_order['treated'] = True
+                        same_pizzas.append(other_pizza_order['content'])
+                quantity = 1 + len(same_pizzas)
+                formatted_order.append({"number": quantity,
+                                        "pizza": pizza_order['content'],
+                                        "price": pizza_order['content'].get_price() * quantity})
```

Comment:
It could be great if we can keep the DéBILs that ordered !
It is used to check if there is no miss click, double order (f5 on post request), etc :D